Web_Development/FrontEnd/Html_Table_Tags/Html_Tables.md

```diff
@@ -0,0 +1,175 @@
+
+## **HTML_Tables**
+
+- The table tag HTML element represents tabular data — that is, information presented in a two-dimensional table comprised of rows and columns of cells containing data. 
+
+## **Table Cells**
+
+- Each table cell is defined by a td tag and closed by td tag.
```

Comment:
Mention the tags enclosed in left & right angle brackets, for eg: _<td>_

Reply:
for eg: `</td>`